local-cli/runIOS/runIOS.js

```diff
@@ -135,7 +135,7 @@ function runOnDevice(selectedDevice, scheme, xcodeProject, configuration){
   });
 }
 
-function buildProject(xcodeProject, udid, scheme, configuration = 'Debug') {
+function buildProject(xcodeProject, udid, scheme, configuration = 'Debug', launchPackager = false) {
```

Comment:
Default function parameter syntax is supported since node 6. I have checked both in node 4 and 5 and it works as expected, since it is transpiled by babel on the CLI entry on _local-cli/cli.js_.